src/services/measure.service.js

```diff
@@ -513,10 +529,13 @@ const systemCodeProgramQuery = program => {
  * @param {Object} query http request query
```

Comment:
this jsdoc needs an update to account for the added flag. Also, can we switch the flag to camelCase? isForQB